proof-systems/src/darlin/marlin/src/constraints/test.rs

```diff
@@ -0,0 +1,206 @@
+#[cfg(test)]
+#[cfg(feature = "circuit-friendly")]
```

Comment:
Well I don't see any reason to have this compilation flag here, as you didn't put it for the whole constraints module, so no additional stuff should be compiled for the test anyway.

Reply:
Flag removed